packages/ethereum-plugin/test/validators/toEmit.test.ts

```diff
@@ -0,0 +1,52 @@
+import { MockProvider } from '@ethereum-waffle/provider'
+import { expect } from 'earljs'
+import { BigNumber, Contract, ContractFactory, Wallet } from 'ethers'
+
+describe('toEmit', function () {
+  this.timeout(10000)
+
+  let events: Contract
+
+  before(async function () {
+    this.timeout(10000)
```

Comment:
Why does it take 10s?

Reply:
It doesn't on my machine, and I'd guess it won't on yours.
Any interaction with MockProvider does take... variable time on CI, though, and I'd rather have slow CI than flaky CI.